AggregateMetrics/AggregateMetrics/AzureWebApp/CacheHelper.cs

```diff
@@ -6,49 +6,96 @@
     using System.Threading.Tasks;
 
     /// <summary>
-    /// Class to contain the one cache for all Gauges 
+    /// Class to contain the one cache for all Gauges.
     /// </summary>
     internal class CacheHelper
     {
+        private static readonly CacheHelper instance = new CacheHelper();
+
+        static CacheHelper() { }
```

Comment:
This static constructor does nothing so it can be removed (there is automatically an implicit one for the creation of the static instance above, which the compiler will create for you, so you don't need to define your own empty one).